src/yachtzee.ts

```diff
@@ -0,0 +1,831 @@
+      throw "Failed to acquire sufficient mojo filters";
+    }
+  }
+  if (horseradishes > 0) {
+    acquire(horseradishes, $item`jumping horseradish`, 60000, true);
+    if (itemAmount($item`jumping horseradish`) < horseradishes) {
+      throw "Failed to acquire sufficient jumping horseradishes";
+    }
+  }
+
+  // Get fishy turns
+  print("Getting fishy turns", "purple");
+  if (haveEffect($effect`Fishy`) + 20 + (havePYECCharge ? 5 : 0) < yachtzeeTurns) {
+    use(1, $item`fish juice box`);
+  }
+  if (!get("fishyPipeUsed")) use(1, $item`fishy pipe`);
+
+  // Final checks
+  if (haveEffect($effect`Fishy`) + (havePYECCharge ? 5 : 0) < yachtzeeTurns) {
+    throw `We only got ${haveEffect($effect`Fishy`)}/${yachtzeeTurns} turns of fishy!`;
+  }
+
+  set("_garboYachtzeeChainDietPlanned", true);
+  return true;
+}
+
+const ignoredSources = [
+  "Orb Prediction",
+  "Pillkeeper Semirare",
+  "Lucky!",
+  "11-leaf clover (untapped potential)",
+];
+const expectedEmbezzlers = sum(
+  embezzlerSources.filter((source: EmbezzlerFight) => !ignoredSources.includes(source.name)),
+  (source: EmbezzlerFight) => source.potential()
+);
+
+function yachtzeePotionProfits(potion: Potion, yachtzeeTurns: number): number {
+  // If we have an unused PYEC then
+  // 1) We if don't have an effect, +5 to gained effect duration
+  // 2) If we already have an effect, +5 to existing effect duration
+  // This means that the first use of a potion that we don't already have an effect of is more valuable than the next use
+  const PYECOffset =
+    have($item`Platinum Yendorian Express Card`) && !get(`expressCardUsed`) ? 5 : 0;
+  const existingOffset = haveEffect(potion.effect()) ? PYECOffset : 0;
+  const extraOffset = PYECOffset - existingOffset;
+  const effectiveYachtzeeTurns = Math.max(
+    Math.min(
+      yachtzeeTurns - haveEffect(potion.effect()) - existingOffset,
+      potion.effectDuration() + extraOffset
+    ),
+    0
+  );
+  const embezzlerTurns = Math.min(
+    expectedEmbezzlers,
+    Math.max(potion.effectDuration() + extraOffset - effectiveYachtzeeTurns, 0)
+  );
+  const barfTurns = Math.max(
+    potion.effectDuration() + extraOffset - effectiveYachtzeeTurns - embezzlerTurns,
+    0
+  );
+  const embezzlerValue = embezzlerTurns > 0 ? potion.gross(embezzlerTurns) : 0;
+  const yachtzeeValue =
+    (effectiveYachtzeeTurns * 2000 * (potion.meatDrop() + 2.5 * potion.familiarWeight())) / 100; // Every 1lbs of lep ~ 2.5% meat drop
+  const barfValue = (barfTurns * baseMeat * turnsToNC) / (turnsToNC + 1);
+
+  return yachtzeeValue + embezzlerValue + barfValue - potion.price(true);
+}
+
+function yachtzeePotionSetup(yachtzeeTurns: number, simOnly?: boolean): number {
+  let totalProfits = 0;
+  const PYECOffset =
+    have($item`Platinum Yendorian Express Card`) && !get(`expressCardUsed`) ? 5 : 0;
+  const excludedEffects = new Set<Effect>();
+
+  if (have($item`Eight Days a Week Pill Keeper`) && !property.getBoolean("_freePillKeeperUsed")) {
```

Comment:
```suggestion
  if (have($item`Eight Days a Week Pill Keeper`) && !get("_freePillKeeperUsed")) {
```

This one's already a native mafia property! use regular get!